roles/database/tasks/main.yml

```diff
@@ -1,8 +1,12 @@
 ---
 - include: install-apt-deps.yml
-  when: initialize_local_db
+  when: fpsd_database_initialize_local_db
+
 - include: initialize-fpsd-db.yml
-  when: initialize_local_db
+  when: fpsd_database_initialize_local_db
```

Comment:
None of the tables are created in that tasklist. That tasklist creates a PostgreSQL role (aka user account), which must exist in order for any non-{postgres,root} users on the same system or any remote clients to write to a database. It is probably possible to either modify our usage of the `postgresql_db` or use `command` instead in order to create the "fpsd" database on the remote server (and for that matter, the creation of the "testfpsd" database, which is in need of either a `when` conditional or the just-mentioned fix).